.eslintrc.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+
+module.exports = {
+  extends: [
+    './node_modules/@airbnb/config-eslint/lib/presets/base.js',
+    './node_modules/@airbnb/config-eslint/lib/presets/next.js',
+    './node_modules/@airbnb/config-eslint/lib/presets/typescript.js',
+    './node_modules/@airbnb/config-eslint/lib/presets/prettier.js',
+    'prettier/@typescript-eslint',
+    'prettier/unicorn',
+  ],
+  parserOptions: {
+    project: path.join(__dirname, "./tsconfig.eslint.json"),
+  },
```

Comment:
Keep running into this error on test files:

```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
   The file does not match your project config: test/connection/callApi/callApi.test.ts.
   The file must be included in at least one of the projects provided.
```

Above line is the fix, but there is no easy way to add it to `nimbus` so I decided to just manually manage all build configs in Git.